### PR TITLE
feat(reference): act-1447 - added capture events to reference pages

### PR DESCRIPTION
**Pull request by TrofimovAnton85**
## Description

- added capture events ( "`Page Viewed`", "`Request Sent`", "`Request Configuration Started`",  "`Customize Request`" ) to new reference pages

Screenshot 2024-07-08 at 15 00 34
Screenshot 2024-07-08 at 15 01 56
Screenshot 2024-07-08 at 15 03 29

## Links:

[JIRA Issue](https://infura.atlassian.net/browse/ACT-1447)

### CODE REVIEW

File: src/lib/segmentAnalytics.js [hunk 1]
```diff
@@ -1,25 +1,45 @@
...
+      ...(userExperience && { user_experience: userExperience })
     });
   }
 };
 
-export const trackPageForSegmentAnalytics = ({ name, path, userId }) => {
+export const trackClickForSegment = ({
+  eventName,
+  clickType,
+  userExperience,
```
Comment (joaniefromtheblock) [inline src/lib/segmentAnalytics.js:17]: @TrofimovAnton85 Just out of curiousity what info does this provide in the analytics. Does it give us insight into loading metrics, or customer satisfaction, etc...?

Comment (TrofimovAnton85) [reply src/lib/segmentAnalytics.js:17]: @joaniefromtheblock  Hey! 
We have several types of events/metrics (they are described in the ticket) that we would like to collect for analytics. Based on them, we can test: what percentage of users use our updated reference pages and how convenient they are for them (do they use the parameter configurator, sending requests).

**Review by joaniefromtheblock — APPROVED** [review]
LGTM

File: src/lib/segmentAnalytics.js (hunk 1 above)
Comment (joaniefromtheblock) [reply src/lib/segmentAnalytics.js:17]: sorry I meant specifically the `userExperience`?

Comment (TrofimovAnton85) [reply src/lib/segmentAnalytics.js:17]: First, we plan to release updated pages for 10% of users (based on the feature flag condition). For them, this `userExperience` metric will take the value “new”, and the remaining 90% users => old pages will be displayed.

**Review by aednikanov — APPROVED** [review]

**Pull request merged**